backoffice/package.json

```diff
@@ -24,30 +23,24 @@
     "@mui/styles": "^5.4.2",
     "@mui/x-data-grid": "^5.5.1",
     "@mui/x-data-grid-pro": "^5.5.1",
-    "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.0.1",
     "@types/lodash": "^4.14.182",
-    "@types/node": "16.4.13",
+    "@types/node": "^20.0.0",
     "@types/react": "^17.0.44",
     "@types/react-dom": "^17.0.9",
     "@types/react-router-dom": "^5.1.8",
-    "axios": "^0.30.0",
+    "axios": "^1.9.0",
     "css-select": "^5.1.0",
     "date-fns": "^4.1.0",
-    "eslint": "^8.23.0",
     "formik": "^2.2.9",
     "inflected": "^2.1.0",
     "jest-canvas-mock": "^2.3.1",
```

Comment:
Could this move to the dev dependencies too.

Reply:
yep 👍 commit: 0c1d896095af7a5c4782309452750dc64b0e0dbd